titan/features/msmw.py

```diff
@@ -0,0 +1,43 @@
+from . import base_feature
```

Comment:
We need a new name for this one still. Something flux or heat bath maybe?

Reply:
I'm confused

Reply:
Sorry, for msmw. Just something to think about if/when we abstract this.